Iphone.html

```diff
@@ -0,0 +1,204 @@
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+  <link rel="stylesheet" href="css/style.css">
+
+  <title>Iphone</title>
+</head>
+
+<body>
+ <!------------------------------------------------startbody----------------------------------------------------------------->
+	   <!------navbar------>
+
+	<!------end navbar------>
+	<!-------------------------carousel---------------------------->
+<div class="container">
+  <div id="myCarousel" class="carousel slide" data-ride="carousel">
+    <!-- Indicators -->
+    <ol class="carousel-indicators">
+      <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+      <li data-target="#myCarousel" data-slide-to="1"></li>
+	  <li data-target="#myCarousel" data-slide-to="2"></li>
+	  <li data-target="#myCarousel" data-slide-to="3"></li>
+	  <li data-target="#myCarousel" data-slide-to="4"></li>
+	  <li data-target="#myCarousel" data-slide-to="5"></li>	  
+    </ol>
+
+    <!-- Wrapper for slides -->
+    <div class="carousel-inner">
+      <div class="item active">
+        <img src="img/21.jpg" alt="" style="width:100%; ">
+      </div>
+
+      <div class="item">
+        <img src="img/22.jpg" alt="" style="width:100%; ">
+      </div>
+    
+      <div class="item">
+        <img src="img/23.jpg" alt="" style="width:100%; ">
+	  </div>
+	  <div class="item">
+        <img src="img/24.jpg" alt="" style="width:100%; ">
+	  </div>
+	  <div class="item">
+        <img src="img/25.jpg" alt="" style="width:100%; ">
+	  </div>
+	  <div class="item">
+        <img src="img/26.jpg" alt="" style="width:100%; ">
+      </div>
+    </div>
+
+    <!-- Left and right controls -->
+    <a class="left carousel-control" href="#myCarousel" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="right carousel-control" href="#myCarousel" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right"></span>
+      <span class="sr-only">Next</span>
+    </a>
+  </div>
+</div>
+<!----------------------------carousel end------------------------->
+		<!----------------------------section------------------------------>
+	
+			 <!----------------------------section end---------------------------------->
+        <main>
+            <div class="card">
+                <div class="card__img-container">
+                    <div class="card__button-container">
+                        <a id="phone-category-button" href="#"
+                                class="card__button">Show Details</a>
+                    </div>
+                    <img src="img/19.jpg" alt="Iphone" class="card__img">
+                </div>
+                <div class="card__title">1000 جنيه</div>
```

Comment:
what is your point of view about using arabic?

Reply:
There is no specific cause but it can be changed.